.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.studio-frontend]
-file_filter = src/data/i18n/locales/po/<lang>.po
+[edx-platform.studio-frontend2]
+file_filter = src/data/i18n/locales/proof_of_concept/<lang>.json
 source_lang = en_US
-type = PO
+type = KEYVALUEJSON
```

Comment:
It would be super freaking cool if I could test this out without merging to master, [but apparently that's beyond the capabilities of the transifex bot for some reason](https://github.com/edx/ecommerce-scripts/pull/86)

Will attempt to make more progress tomorrow with judicious use of my admin rights. stay tuned.